resources/views/Widgets/Basket/BasketTotalsWidget.json

```diff
@@ -7,6 +7,15 @@
     "position": 300,
     "widgetClass": "Ceres\\Widgets\\Basket\\BasketTotalsWidget",
     "settings": {
+        "customClass": {
```

Comment:
Hey @Xnaff, vielen Dank für die Arbeit, die du in die Weiterentwicklung des Produktes gesteckt hast.
Wir werden die Erweiterung gerne übernehmen. 
Allerdings gibt es noch ein Paar stellen die im Code vereinfacht werden können.
An dieser Stelle (im JSON der Widgets) sollte der Code durch ein include von `customClass` gelöst werden, um Zeilen einzusparen.

Erstelle eine include Datei wie hier:
Ceres/resources/views/Widgets/Helper/Includes/spacing.inc.json

`customClass.inc.json`

dort kannst du dein customClass JSON hinterlegen und es im Widget JSON importieren
"customClass":"@include(customClass)"
Bei `npm run collectWidgets` werden dann die includes aufgelöst.

Beste Grüße Maxi

Reply:
Hi @maxiroellplenty  ich habe die `customClass` als include gelöst